packages/connectivity_plus/connectivity_plus/lib/connectivity_plus.dart

```diff
@@ -10,6 +10,8 @@ import 'package:connectivity_plus_platform_interface/connectivity_plus_platform_
 export 'package:connectivity_plus_platform_interface/connectivity_plus_platform_interface.dart'
     show ConnectivityResult;
 
+export 'src/connectivity_plus_linux.dart';
+
```

Comment:
The newest version of the connectivity_plus package does not work after the 3.0.0 version release. Shouldn't you have `export 'src/connectvity_plus_web.dart'` here?

Reply:
See https://github.com/fluttercommunity/plus_plugins/issues/1270

Reply:
But yes, probably the export web is missing here. Please submit a PR or an issue if you can!

Reply:
https://github.com/fluttercommunity/plus_plugins/pull/1278